CONTRIBUTING.md

```diff
@@ -0,0 +1,121 @@
+# Contributing
+
+Welcome! We're excited that you're interested in contributing. Below are some
+basic guidelines.
+
+## Workflow
+
+Dskit follows a standard GitHub pull request workflow. If you're unfamiliar
+with this workflow, read the very helpful [Understanding the GitHub
+flow][github-flow] guide from GitHub.
+
+[github-flow]: https://guides.github.com/introduction/flow/
+
+You are welcome to create draft PRs at any stage of readiness - this can be
+helpful to ask for assistance or to develop an idea. But before a piece of work
+is finished it should:
+
+* Be organised into one or more commits, each of which has a commit message
+  that describes all changes made in that commit ('why' more than 'what' - we
+  can read the diffs to see the code that changed).
+
+* Each commit should build towards the whole - don't leave in back-tracks and
+  mistakes that you later corrected.
+
+* Have unit tests for new functionality or tests that would have caught the bug
+  being fixed.
+
+* Include a CHANGELOG message if users of Dskit need to hear about what you
+  did.
+
+## Testing und Linting the Code
+
+To lint the code:
+
+```bash
+make lint
+```
+
+To run the unit tests suite:
+
+```bash
+make test
+```
+
+### Dependency management
+
+We use [go modules] to manage dependencies on external packages.  This requires
+a working Go environment with version 1.16 or greater, git and [bzr] installed.
+
+[Go modules]: https://golang.org/cmd/go/#hdr-Modules__module_versions__and_more
+[bzr]: http://wiki.bazaar.canonical.com/Download
```

Comment:
:) this looks old

Reply:
Agreed and removed, haven't heard about bazaar in a while :joy: